Lib/http/server.py

```diff
@@ -902,6 +937,24 @@ def guess_type(self, path):
             return guess
         return 'application/octet-stream'
 
+    def get_range(self):
```

Comment:
I also think that `parse_range` is more explicit.

Reply:
Have changed that.